jobs/kubelet/templates/bin/ensure_kubelet_up_and_running.erb

```diff
@@ -2,11 +2,76 @@
 
 [ -z "$DEBUG" ] || set -x
 
+CERT_DIR=/var/lib/kubelet/pki
```

Comment:
Should we should specify `--cert-dir` on the kubelet to make sure the credentials stay in `/var/vcap/jobs/kubelet/config`?

Reply:
So long as `/var/vcap/jobs/*` isn't a persistent disk, that would work fine. If it's persistent, however, it creates problems during VM recovery as underlying IP and DNS names may no longer match the ones specified in the generated certificate SANs.

I kept it in the default location _only_ because I knew it to not be persistent.

Reply:
`/var/vcap/jobs` shouldn't be persistent. I believe `/var/vcap/store` is the only place persistent disks get mounted. It might be worth using `/var/vcap/jobs/kubelet/config` for consistency.